src/app/apiApp.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import express from "express";
+
+export default function apiApp(sourcecredDirectory: string) {
```

Comment:
can we either:
- remove `node bin/sourcecred.js start`
- or refactor it to depend on this code?

Reply:
The new behavior of `node ./bin/sourcecred.js start` will be to start a prod server. I'll implement that presently.